core/server/src/test/java/alluxio/master/AlluxioMasterRestServiceHandlerTest.java

```diff
@@ -0,0 +1,237 @@
+    mMaster = mock(AlluxioMaster.class);
+    mContext = mock(ServletContext.class);
+    Journal journal = mock(Journal.class);
+    mBlockMaster = new BlockMaster(journal);
+    when(mMaster.getBlockMaster()).thenReturn(mBlockMaster);
+    when(mContext.getAttribute(MasterUIWebServer.ALLUXIO_MASTER_SERVLET_RESOURCE_KEY))
+        .thenReturn(mMaster);
+    when(mMaster.getMasterAddress()).thenReturn(new InetSocketAddress("localhost", 8080));
+    when(mContext.getAttribute(MasterUIWebServer.ALLUXIO_MASTER_SERVLET_RESOURCE_KEY))
+        .thenReturn(mMaster);
+    mHandler = new AlluxioMasterRestServiceHandler(mContext);
+  }
+
+  @Test
+  public void testGetConfiguration() {
+    Response response = mHandler.getConfiguration();
+    assertNotNull("Response must be not null!", response);
+    assertNotNull("Response must have a entry!", response.getEntity());
+    assertTrue("Entry must be a SortedMap!", (response.getEntity() instanceof SortedMap));
+    SortedMap<String, String> entry = (SortedMap<String, String>) response.getEntity();
+    assertFalse("Properties Map must be not empty!", (entry.isEmpty()));
+  }
+
+  @Test
+  public void testGetRpcAddress() {
+    Response response = mHandler.getRpcAddress();
+    assertNotNull("Response must be not null!", response);
+    assertNotNull("Response must have a entry!", response.getEntity());
+    assertEquals("Entry must be a String!", String.class, response.getEntity().getClass());
+    String entry = (String) response.getEntity();
+    assertFalse("Properties Map must be not empty!", (entry.isEmpty()));
+  }
+
+  @Test
+  public void testGetMetrics() {
+    final int FILES_PINNED_TEST_VALUE = 100;
+    String filesPinnedProperty =
+        MetricsSystem.getMasterMetricName(FileSystemMaster.Metrics.FILES_PINNED);
+    Gauge<Integer> filesPinnedGauge = new Gauge<Integer>() {
+      @Override
+      public Integer getValue() {
+        return FILES_PINNED_TEST_VALUE;
+      }
+    };
+    MetricSet mockMetricsSet = mock(MetricSet.class);
+    Map<String, Metric> map = new HashMap<>();
+    map.put(filesPinnedProperty, filesPinnedGauge);
+
+    when(mockMetricsSet.getMetrics()).thenReturn(map);
+    MetricsSystem.METRIC_REGISTRY.registerAll(mockMetricsSet);
+
+    Response response = mHandler.getMetrics();
+    assertNotNull("Response must be not null!", response);
+    assertNotNull("Response must have a entry!", response.getEntity());
+    assertTrue("Entry must be a SortedMap!", (response.getEntity() instanceof SortedMap));
+    SortedMap<String, Long> entry = (SortedMap<String, Long>) response.getEntity();
+    assertFalse("Properties Map must be not empty!", (entry.isEmpty()));
+    assertTrue("Map must contains key " + filesPinnedProperty + "!",
+        entry.containsKey(filesPinnedProperty));
+    assertEquals(FILES_PINNED_TEST_VALUE, entry.get(filesPinnedProperty).longValue());
+  }
+
+  @Test
+  public void testGetStartTimeMs() {
+    Response response = mHandler.getStartTimeMs();
```

Comment:
We should mock the mMaster's `getStartTime()` method to test a more interesting value here